pennylane/beta/plugins/default_tensor.py

```diff
@@ -523,29 +542,29 @@ def _ev_exact(self, obs_nodes, wires):
             ket_and_observable_node = tn.contract_between(obs_node, ket_and_observable_node)
         return tn.contract_between(bra, ket_and_observable_node).tensor
 
-    def _ev_mps(self, obs_nodes, wires):
+    def _ev_mps(self, obs_nodes, registers):
         r"""Expectation value of observables on specified wires using a MPS representation.
 
          Args:
             obs_nodes (Sequence[tn.Node]): the observables as TensorNetwork Nodes
-            wires (Sequence[Sequence[int]]): measured subsystems for each observable
+            registers (Sequence[Sequence[int]]): indices of measured wires for each observable
          Returns:
             complex: expectation value :math:`\expect{A} = \bra{\psi}A\ket{\psi}`
         """
-        if any(len(wires_seq) > 2 for wires_seq in wires):
+        if any(len(wires_seq) > 2 for wires_seq in registers):
             raise NotImplementedError(
                 "Multi-wire measurement only supported for nearest-neighbour wire pairs."
             )
-        if len(obs_nodes) == 1 and len(wires[0]) == 1:
+        if len(obs_nodes) == 1 and len(registers[0]) == 1:
             # TODO: can measure multiple local expectation values at once,
             # but this would require change of `expval` behaviour and
             # refactor of `execute` logic from parent class
-            expval = self.mps.measure_local_operator(obs_nodes, wires[0])[0]
+            expval = self.mps.measure_local_operator(obs_nodes, registers[0])[0]
```

Comment:
another place where things would likely break if `registers` is not a consecutive, zero-first list, since we are calling a function from an external library